src/Effect/App.js

```diff
@@ -0,0 +1,66 @@
+import { applyMiddleware, createStore } from 'redux'
+import { combineEpics, createEpicMiddleware } from 'redux-observable'
+import { compose, flip, uncurryN } from 'ramda'
+import { render } from 'react-dom'
+import { tryCatch, chain } from 'crocks'
+import App from '../Component/App'
+import React from 'react'
+import rootReducer from '../State'
+
+
+// rootEpic :: (Observable Action, Observable State) -> Observable Action
+export const rootEpic = combineEpics()
+
+
+// configureStore :: () -> Result Error Redux.Store
+export const configureStore = tryCatch(() => {
+  const env = process.env.NODE_ENV || 'development'
```

Comment:
better fallback to `prod` env by default IMO (safer). `dev` env should be an opt-in config ;)